roles/ceph-mon/tasks/docker/main.yml

```diff
@@ -125,4 +125,4 @@
       - item.stat.exists == true
   when:
     - inventory_hostname == groups[mon_group_name]|last
-    - ceph_release_num.{{ ceph_stable_release }} > ceph_release_num.jewel
+    - not ceph_docker_image_tag.find('jewel') != -1 or not ceph_docker_image_tag.find('ceph-2') != -1
```

Comment:
Yes, images name standardization is pretty bad at the moment. It'll be nice to have something a bit more solid. At least, something we can rely on without having to go back in a couple of months to update it...